account_avatax_exemption/models/exemption.py

```diff
@@ -0,0 +1,332 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+
+class ExemptionRule(models.Model):
+    _name = "exemption.code.rule"
+    _description = "Avatax Custom Rules"
+
+    name = fields.Char(index=True, default=lambda self: self.env._("New"))
```

Comment:
```suggestion
    name = fields.Char(index="trigram", default=lambda self: self.env._("New"))
```
change `index=True` to `index='gin'` in: https://github.com/odoo/odoo/pull/83015/commits/12c655b497fbe101e94c9ecb68ab0b56c287d0f8#diff-695321b9df7129604a59fbc9bff34f97ba90b3394aa1166ca24e36b27389ae82

then from `index='gin'` to `index='trigram'` in: https://github.com/odoo/odoo/commit/a1904aa6f62acbd3d8e7d4cc873692e20edb37fd#diff-695321b9df7129604a59fbc9bff34f97ba90b3394aa1166ca24e36b27389ae82